gitlab-pipeline/stage/test.yml

```diff
@@ -251,8 +251,13 @@ test:backend-integration:azblob:enterprise:
     - df -h . | tail -1 | awk '{system("hdparm -tT "$1);}'
 
     # Get and install the integration test requirements
-    - apk add $(cat ${WORKSPACE}/integration/tests/requirements/apk-requirements.txt)
-    - pip install -r ${WORKSPACE}/integration/tests/requirements/python-requirements.txt
+    - if [ -d ${WORKSPACE}/integration/tests/requirements-python ]; then
+    -   apk add $(cat ${WORKSPACE}/integration/tests/requirements-system/apk-requirements.txt)
+    -   pip install -r ${WORKSPACE}/integration/tests/requirements-python/python-requirements.txt
```

Comment:
Shouldn't we have separate if conditions for these two lines?

And smart, btw. 🙂

Reply:
I think it is pretty safe to assume that if we have migrated to separate directories both shall exist. I couldn't understand someone porting this half-way to a release branch, for example. And if that happens better to just error and fix in `integration`.